internal/devbox/secrets.go

```diff
@@ -9,41 +9,36 @@ import (
 	"go.jetpack.io/pkg/envvar"
 )
 
-type secrets struct {
-	envsec.Envsec
-	EnvName string
-}
-
-func (d *Devbox) Secrets(ctx context.Context) (*secrets, error) {
-	envsecInstance := envsec.Envsec{
+func (d *Devbox) UninitializedSecrets(ctx context.Context) *envsec.Envsec {
+	return &envsec.Envsec{
 		APIHost: build.JetpackAPIHost(),
 		Auth: envsec.AuthConfig{
 			ClientID: envvar.Get("ENVSEC_CLIENT_ID", build.ClientID()),
 			Issuer:   envvar.Get("ENVSEC_ISSUER", build.Issuer()),
 		},
 		IsDev:      build.IsDev,
 		Stderr:     d.stderr,
+		Store:      &jetstore.JetpackAPIStore{},
 		WorkingDir: d.ProjectDir(),
+		EnvID: envsec.EnvID{
+			EnvName: d.environment,
```

Comment:
is it possible for `d.environment` to not be set? If so, could we add a validation check in this function?

Reply:
it's already validated in `devbox.Open`